### PR TITLE
Crncc 1461 vs cms welsh 1 6 emails

### PR DESCRIPTION
**Pull request by paddyduff31**

### CODE REVIEW

File: Dte.Common/Services/ContentfulService.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            {
+                link = request.Link,
+                firstName = ToTitleCase(request.FirstName, selectedLocale)
+            };
+
+            var template = Handlebars.Compile(htmlTemplate);
+            return template(data);
+        }
+
+        private static string ToTitleCase(string input, CultureInfo cultureInfo)
```
Comment (con-nihr) [inline Dte.Common/Services/ContentfulService.cs:59]: Nice

File: Dte.Common/Http/ContentfulClient.cs
Comment (con-nihr) [inline Dte.Common/Http/ContentfulClient.cs:1]: Probably best to name this file and the class `ContentfulServiceCollectionExtensions.cs`.

FYI, some projects use a trick by adding these extensions to the `Microsoft.Extensions.DependencyInjection` namespace.

That way you get Intellisense directly in your Startup.cs when you add a reference to the library.

e.g.
...
AddContentfulServices()
AddRazorPages()
AddHttpClient()
...etc

see https://github.com/AutoMapper/AutoMapper.Extensions.Microsoft.DependencyInjection/blob/master/src/AutoMapper.Extensions.Microsoft.DependencyInjection/ServiceCollectionExtensions.cs

Not suggesting you need to do this, but something to be aware of.

**Review by con-nihr — CHANGES_REQUESTED** [review]
Love it. Well done.

Do you need to incorporate this in to the api and event handlers now?

**Comment by paddyduff31** [comment]
Yes, it now just has to be implemented but I need access to the package services so I need to get this deployed first before updating the other 2

File: Dte.Common/Http/ContentfulClient.cs
[the remark this answers is not in the document]
Comment (paddyduff31) [reply Dte.Common/Http/ContentfulClient.cs:1]: Apologies, I just did the class name itself!!!

Done

**Comment by paddyduff31** [comment]
Added BaseUrl to the request so common knows the env we are in

**Review by con-nihr — APPROVED** [review]

**Pull request merged**